topics/variant-analysis/tutorials/trio-analysis/tutorial.md

```diff
@@ -0,0 +1,438 @@
+---
+layout: tutorial_hands_on
+
+title: Trio Analysis using Synthetic Datasets from RD-Connect GPAP
+zenodo_link: 'https://doi.org/10.5281/zenodo.6483454'
+questions:
+- How do you import data from the EGA?
+- How to download files with HTSGET in Galaxy?
+- How do you pre-process VCFs?
+- How do you identify causative variants?
+objectives:
+- Requesting DAC access and importing data from the EGA.
+- Pre-process VCFs using regular expressions.
+- Use annotations and phenotype information to find the causative variant(s).
+time_estimation: 2H
+key_points:
+- Downloading whole datasets with HTSGET is safe and easy with galaxy.

```

Comment:
Please capitalize all occurrences of "Galaxy"

```suggestion
- Downloading whole datasets with HTSGET is safe and easy with Galaxy.

```

Reply:
Fixed. 76a329e7416e82ac840f51baaf63cb82a4358a73